docs/book/development.md

```diff
@@ -0,0 +1,129 @@
+# vSphere CSI Driver - Driver Development
+
+## How to build this project
+
+- Fork the project on [github](https://github.com/kubernetes-sigs/vsphere-csi-driver)
```

Comment:
Can we change this to `create a fork of [https://github.com/kubernetes-sigs/vsphere-csi-driver](https://github.com/kubernetes-sigs/vsphere-csi-driver)`

Reply:
Fixed.